src/Youtube.js

```diff
@@ -638,7 +638,7 @@ THE SOFTWARE. */
     return (e === 'video/youtube');
   };
 
-  var _isOnMobile = videojs.browser.IS_IOS || useNativeControlsOnAndroid();
+  var _isOnMobile = videojs.browser.IS_IOS || videojs.browser.IS_ANDROID;
```

Comment:
it may be a good idea to move this towards the top of the file where all the other scope level globals are. Seems weird to have this mid-file. LGTM other than that though.